packages/dx-react-grid/docs/reference/table-column-resizing.md

```diff
@@ -13,6 +13,7 @@ A plugin that manages table column widths.
 Name | Type | Default | Description
 -----|------|---------|------------
 columnWidths? | Array&lt;[TableColumnWidthInfo](#tablecolumnwidthinfo)&gt; | | Specifies column widths.
+minColumnWidth? | number | `45` for [Bootstrap3](https://www.npmjs.com/package/@devexpress/dx-react-grid-bootstrap3); `55` for [Bootstrap4](https://www.npmjs.com/package/@devexpress/dx-react-grid-bootstrap4); `40` for [Material UI](https://www.npmjs.com/package/@devexpress/dx-react-grid-material-ui); | Specifies minimal column width.
```

Comment:
Maybe we should change property name to `minColumnsWidth`? It shows that minimal width defines for each column.

Reply:
@MaximKudriavtsev, in this case, it is pretty clear that 'minColumnWidth' option is used for all columns. in addition, other our properties use the singular form and defined for all columns/rows.